lib/fastlane/actions/git_pull.rb

```diff
@@ -2,14 +2,23 @@ module Fastlane
   module Actions
     class GitPullAction < Action
       def self.run(params)
-        command = [
+        commands = []
+
+        unless params[:only_tags]
+          commands.concat [
+            'git',
+            'pull',
+            '&&'
+          ]
+        end
+
+        commands.concat [
           'git',
-          'pull',
+          'fetch',
           '--tags'
         ]
 
-        Actions.sh(command.join(' '))
-        Helper.log.info 'Sucesfully pulled from remote.'
```

Comment:
Because that was the last command executed, that's the result I got in the tests :-/ I couldn't figure out another way of getting which commands were being run if I left that line. It also seemed like not super valuable (considering you can still see that the action is being run).
What do you think?